runtime/src/lib.rs

```diff
@@ -95,6 +95,9 @@ impl sdk::Runtime for Runtime {
     );
 
     fn trusted_policy_signers() -> Option<TrustedPolicySigners> {
+        if option_env!("OASIS_UNSAFE_SKIP_KM_POLICY") == Some("1") {
```

Comment:
The other places just check whether the env variable is present, but this works too.